README.md

```diff
@@ -15,7 +15,7 @@ Jackson datatype module for [Vavr](http://vavr.io/) library
 <dependency>
   <groupId>io.vavr</groupId>
   <artifactId>vavr-jackson</artifactId>
-  <version>0.9.0</version>
+  <version>1.0.1</version>
```

Comment:
In the "Usage" paragraph of `README` we use [latest release](https://oss.sonatype.org/content/repositories/releases/io/vavr/vavr-jackson/) version of `vavr-jackson`, in "developer versions" we use [latest snapshot](https://oss.sonatype.org/content/repositories/snapshots/io/vavr/vavr-jackson/) version. Now it is `1.0.0-SNAPSHOT`, see [pom.xml at master branch](https://github.com/vavr-io/vavr-jackson/blob/master/pom.xml#L12)